packages/replayio/src/utils/prompt/prompt.ts

```diff
@@ -38,6 +40,7 @@ export async function prompt({
         case "\x03":
           // \x03 is Ctrl+C (aka "End of text")
           // https://donsnotes.com/tech/charsets/ascii.html
+          onExit?.();
```

Comment:
I wonder if we can handle this in a broader way by listening for "SIGINT" ?

I think what you have here _works_ but it's also kind of narrowly scoped.

Reply:
> I wonder if we can handle this in a broader way by listening for "SIGINT" ?

I wanted to do that at first but in raw mode SIGINT is simply not dispatched here. The process gets closed **only** because we call `process.exit()` on our own from within this handler~.

Reply:
> I wanted to do that at first but in raw mode SIGINT is simply not dispatched here.

Ah, ok.